enarx-prstate

```diff
@@ -2,6 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from github import Github
+import datetime
```

Comment:
This is never used.

Reply:
Thought I would need it for the comparison of two `datetime` objects in the script, but I guess not!